### PR TITLE
[TASK] update es6 and check for access keys

### PR DESCRIPTION
**Pull request by AdamCmiel**
mv config to the root of the project
for the built file to access it as well

### CODE REVIEW

**Comment by AdamCmiel** [comment]
now the client fails when you don't give it access keys instead of the crypto library throwing that it can't sign nothing

File: src/Client.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import bb from 'bluebird'
+import { DOMAIN } from '../config'
+
+const http = bb.promisifyAll(superagent)
+
+export default class Client {
+
+  constructor(options) {
+
+    if (options.accessKeyId === undefined || options.secretAccessKey === undefined) {
```
Comment (stevenzeiler) [inline src/Client.js]: much better

File: package.json [hunk 2]
```diff
@@ -2,15 +2,18 @@
   "name": "anypay-client",
   "version": "1.3.0",
```
Comment (stevenzeiler) [inline package.json]: let's bump the version to 1.3.1

**Comment by stevenzeiler** [comment]
:+1:

**Pull request merged**